packages/js/explat/src/error.ts

```diff
@@ -26,7 +26,7 @@ export const logError = (
 		if ( isDevelopmentMode ) {
 			console.error( '[ExPlat] ', error.message, error ); // eslint-disable-line no-console
 		} else {
-			if ( ! window.wcTracks?.isEnabled ) {
+			if ( ! window.wcTracks?.isEnabled && ! window?._wca?.push ) {
```

Comment:
Can we do this? You'll need the `import` too of course.
```suggestion
			if ( ! canTrack ) {
```